.gitignore

```diff
@@ -1,8 +1,33 @@
-_build/
```

Comment:
No problem with adding a leading `/` to all directories, but the other changes should be reverted.

For example I have never seen a `/tmp/` here so it should not be ignored. The comment lines are imho somewhat unhelpful especially since the project specific ones are documented with a plain "misc" so I would just remove them altogether.